common/changes/@microsoft/rush/path-spaces_2018-12-20-21-57.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "comment": "Fix install-run-rush handling of spaces in paths",
+      "packageName": "@microsoft/rush",
+      "type": "none"
```

Comment:
Shouldn't this be patch?

Reply:
It was automatically set to none, I'm guessing because there's already a patch update pending (Rush isn't published automatically)

Reply:
Rush uses lockstep versioning, so the version bump is determined by the config file that governs the next release, not by the person running `rush change`.  I agree that `"type": "none"` doesn't convey this very clearly.